elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/cache/CaffeineCache.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.datastores.aggregation.cache;
+
+import com.yahoo.elide.datastores.aggregation.query.QueryResult;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+
+/**
+ * A basic local-only cache.
+ */
+public class CaffeineCache implements Cache {
+    public static final int DEFAULT_CACHE_SIZE = 1024;
```

Comment:
What is the size in terms of bytes?  Entries?  How about change the variable name to include units.